bridge/cypress/integration/uniform-integrations.spec.ts

```diff
@@ -589,6 +594,10 @@ describe('Add control plane subscription default requests', () => {
   it('should not show webhook form', () => {
     cy.get('ktb-webhook-settings').should('not.exist');
   });
+
+  it('should show danger zone on edit page', () => {
+    cy.get('ktb-danger-zone').should('not.exist');
```

Comment:
Could you either move the danger-zone check to the related page object or to the global commands?

Reply:
Could you use the page object instead? `uniformPage.assertDangerZoneExists(false)`